File: CST2335_Final-milestone_2/app/src/main/java/com/example/cst2335_final/MainActivity.java
package com.example.cst2335_final;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.inputmethod.InputMethodManager;
import android.widget.ArrayAdapter;
import android.widget.BaseAdapter;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ListView;
import android.widget.ProgressBar;

import com.example.cst2335_final.adapters.SearchListAdapter;
import com.example.cst2335_final.beans.SearchItem;

import java.util.ArrayList;

/**
 * Main Activity for application
 */
public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";
    private BaseAdapter searchListAdapter;
    private ArrayList<SearchItem> items = new ArrayList<>();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Intent nextPage =new Intent(this, GuardianSearchActivity.class);
        Button guardianSearchBtn = findViewById(R.id.guardianSearchBtn);
        guardianSearchBtn.setOnClickListener( (click) -> startActivity(nextPage));


    }
}

File: ViewPager2/app/src/main/java/com/example/viewpager2/MainActivity.kt
package com.example.viewpager2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() , MyFragment.callback {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun oneopen() {
        Log.d("TAG", "oneopen: ")
    }

    override fun twoopen() {
        Log.d("TAG", "twoopen: ")
    }

    override fun threeopen() {
        Log.d("TAG", "threeopen: ")
    }

}